Tools/scripts/build_options.py

```diff
@@ -184,6 +184,7 @@ def __init__(self,
     Feature('Plane', 'DEEPSTALL', 'HAL_LANDING_DEEPSTALL_ENABLED', 'Enable Deepstall Landing', 0, None),
     Feature('Plane', 'QAUTOTUNE', 'QAUTOTUNE_ENABLED', 'Enable QuadPlane Autotune mode', 0, "QUADPLANE"),
     Feature('Plane', 'PLANE_BLACKBOX', 'AP_PLANE_BLACKBOX_LOGGING', 'Enable blackbox logging', 0, None),
+    Feature('Plane', 'AP_TUNING', 'AP_TUNING_ENABLED', 'Enable two-switch parameter tuning', 0, None),
```

Comment:
then change this so as to not confuse users
```suggestion
    Feature('Plane', 'Transmitter based tuning', 'AP_TUNING_ENABLED', 'Enable TX based tuning parameter adjustments', 0, None),
```

Reply:
Done, thanks.